crates/autopilot/src/arguments.rs

```diff
@@ -203,6 +203,24 @@ pub struct Arguments {
         value_parser = shared::arguments::duration_from_seconds,
     )]
     pub solve_deadline: Duration,
+
+    /// How much of the order's surplus should be taken as a protocol fee.
+    #[clap(
+        long,
+        env,
+        default_value = "0",
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub protocol_fee_factor: f64,
+
+    /// Cap protocol fee with a percentage of the order's volume.
+    #[clap(
+        long,
+        env,
+        default_value = "0",
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub protofol_fee_volume_cap_factor: f64,
```

Comment:
Given that there will be likely a number of more parameters related to the fee, can we create a sub argument struct for fee related fields (instead of adding a lot more value to the top level list). Eventually we should move towards a `.toml` based config here as well, but I don't see this happening very soon.

Reply:
Fixed.